docs/source/_templates/index.html

```diff
@@ -1,1730 +1,215 @@
-</script>
-
-</body>
-</html>
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title>MegaQC</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="_static/css/main.css" />
+        <script src="https://kit.fontawesome.com/d600571087.js" crossorigin="anonymous"></script>
+	</head>
+	<body class="homepage">
+		<div id="page-wrapper">
+
+			<!-- Header -->
+				<div id="header-wrapper" class="wrapper">
+					<div id="header">
+
+						<!-- Logo -->
+							<div id="logo">
+								<h1><a href="index.html"><img src="_static/images/MegaQC_logo_darkbg.svg" title="MegaQC"></a></h1>
+								<p>Collect and visualise data from multiple MultiQC reports</p>
+							</div>
+
+						<!-- Nav -->
+							<nav id="nav">
+								<ul>
+									<li class="current"><a href="index.html">Home</a></li>
+									<li><a href="docs/index.html" title="Read the MegaQC documentation">Docs</a></li>
```

Comment:
I was always using `{{ pathto("docs/index") }}` or whatever, but hardcoding it should be fine as well.